indexing-hadoop/src/test/java/io/druid/indexer/hadoop/DatasourceInputFormatTest.java

```diff
@@ -145,11 +230,14 @@ public void testGetSplitsCombineInTwo() throws Exception
         Sets.newHashSet(segments.get(0), segments.get(2)),
         Sets.newHashSet((((DatasourceInputSplit) splits.get(0)).getSegments()))
     );
+    Assert.assertArrayEquals(new String[]{"s2", "s1", "s3"}, splits.get(0).getLocations());
 
     Assert.assertEquals(
         Sets.newHashSet(segments.get(1)),
         Sets.newHashSet((((DatasourceInputSplit) splits.get(1)).getSegments()))
     );
+    System.out.println(Arrays.toString(splits.get(1).getLocations()));
```

Comment:
pls remove, or, i guess you need some IDE automation to remove System.out.println(..)s :-)